metron-platform/metron-writer/src/main/java/org/apache/metron/writer/hdfs/HdfsWriter.java

```diff
@@ -109,10 +119,12 @@ public BulkWriterResponse write(String sourceType
                 (String)configurations.getSensorConfig(sourceType).getOrDefault(IndexingConfigurations.OUTPUT_PATH_FUNCTION_CONF, ""),
                 message
         );
+        LOG.trace("Writing message {} to path: {}", message.toJSONString(), path);
         SourceHandler handler = getSourceHandler(sourceType, path, configurations);
         handler.handle(message, sourceType, configurations, syncPolicyCreator);
       }
     } catch (Exception e) {
+      LOG.error("HdfsWriter encountered error writing", e);
```

Comment:
This log statement is useful.  Can you add the sourceType, number of messages, and output path to the log statement?  That context is going to be helpful when debugging an issue.